### PR TITLE
Typo in README.md #487

### PR DESCRIPTION
**Pull request by KelvinEuclides**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**TODOs**:

- [x] squashed commits
- [ ] includes documentation
- [ ] adds unit tests

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by cpanato** [comment]
@KelvinEuclides can you rebase your pr, please and fix the conflicts?

**Comment by cpanato** [comment]
also need to sign the CLA

**Review by jayesh-srivastava — CHANGES_REQUESTED** [review]
I guess you shouldn't be leaving unnecessarily blank lines. What say?

**Comment by KelvinEuclides** [comment]
Ok let me fix that real fast

**Review by jayesh-srivastava — CHANGES_REQUESTED** [review]
I guess you should 'squash' the commits into one. Three different commits for the same file aren't needed.

**Comment by jayesh-srivastava** [comment]
Also, you need to sign the CLA before we can proceed further with the tests.

**Comment by sayantani11** [comment]
Please sign the CLA

File: README.md [hunk 1]
```diff
@@ -44,37 +40,34 @@ This provider's versions are able to install and manage the following versions o
...
 
-To set up the development environment, checkout the [development guide].
+To set up the development environment check out the [development guide].
 
-In the interest of getting new people involved, we have issues marked as [`good first issue`][good_first_issue]. Although
-these issues have a smaller scope but are very helpful in getting acquainted with the codebase.
-For more, see the [issue tracker]. If you're unsure where to start, feel free to reach out to discuss.
+In the interest of getting new people involved, we have issues marked as [`good first issue`][good_first_issue].
+
+Although These issues have a smaller scope but are very helpful in getting acquainted with the codebase.
```
Comment (cpanato) [inline README.md:61]:
```suggestion
Although these issues have a smaller scope but are very helpful in getting acquainted with the codebase.
```

**Comment by KelvinEuclides** [comment]
Good morning i have already signed both documents

**Comment by KelvinEuclides** [comment]
image

**Comment by sayantani11** [comment]
@KelvinEuclides 
```
7. My pull request still doesn't pass!
If you haven't yet, post a comment ("I signed it") to the PR—this is needed for the system to recheck your contribution authorization.

Each commit in the pull request needs to pass. If you have commits that didn't attribute to a Github username, or to a different user that hasn't signed the CLA, that will fail that commit and all subsequent commits.
```

**Comment by sayantani11** [comment]
@KelvinEuclides Just for the sake of it, can you also try "I signed it"

**Comment by KelvinEuclides** [comment]
I signed it

**Comment by jayesh-srivastava** [comment]
@KelvinEuclides What @sayantani11 means to say is that you can just comment directly here the quote, I signed it , with double quotations marks. I guess that'll work.

**Comment by KelvinEuclides** [comment]
"I signed it"

**Review by cpanato — COMMENTED** [review]
/ok-to-test

**Comment by sayantani11** [comment]
Hmm @KelvinEuclides Can you squash the commits?

**Comment by KelvinEuclides** [comment]
Sorry ment to squash

File: README.md (hunk 1 above)
Comment (cpanato) [reply README.md:61]: @KelvinEuclides please review the comments and also squash the commits

**Comment by cpanato** [comment]
humm sorry, just saw that it is still missing some changes

/lgtm cancel
/approve cancel

File: README.md (hunk 1 above)
Comment (cpanato) [reply README.md:61]: @KelvinEuclides gently ping on this

**Comment by richardcase** [comment]
@KelvinEuclides - just checking in, any chance of squashing commits on this?

**Comment by cpanato** [comment]
@KelvinEuclides gently ping to review the feedback and rebase.

**Pull request closed without merging**